src/engine/schema.rs

```diff
@@ -2307,6 +2303,11 @@ fn generate_schema(c: &Configuration) -> Result<HashMap<String, NodeType>, Error
         NodeType::new("Query".to_string(), TypeKind::Object, query_props),
     );
 
+    nthm.insert(
```

Comment:
@maldrake : what is this type for?

Reply:
The new version of Juniper supports GraphQL subscriptions. The `RootNode` type has been expanded to include a root Query, root Mutation, and root Subscription. It's required for the root node, even if subscriptions are unused, so we include an empty root node with no children.